Engine/Module/ModuleFileSystem.cpp

```diff
@@ -44,9 +44,14 @@ ModuleFileSystem::~ModuleFileSystem()
 }
 
 char* ModuleFileSystem::Load(const char* file_path, size_t & size) const
-{
+{ 
 	SDL_RWops *rw = SDL_RWFromFile(file_path, "rb");
-	if (rw == NULL) return NULL;
+	if (rw == NULL)
+	{
+		size = 0;
+		return NULL;
+
+	}
```

Comment:
isn't it better to have an assert? I think Carlos said it :/

Reply:
Depends on the behavior we want. An asserts will cause a crash and we may want to don't crash in case we want to allow trying loading files that doesn't exists and control it in run time.

Reply:
Depends on the behavior we want. An asserts will cause a crash and we may don't want to crash in case we want to allow trying loading files that doesn't exists and control it in run time.

Reply:
Perfect!